dev_guide/builds.adoc

```diff
@@ -22,13 +22,15 @@ runnable images to be used on OpenShift. There are three build strategies:
 
 == Defining a buildConfig
 
-A `*buildConfig*` describes a single build definition and a
-set of link:#triggers[triggers] for when a new build should be created.
+A build configuration describes a single build definition and a set of
+link:#triggers[triggers] for when a new build should be created.
 
-A `*buildConfig*` is a REST object, which can be used in a POST to the API server to
-create a new instance. The following example `*buildConfig*` results in a new build
-every time a Docker image tag or the source code changes:
+A build configuration is defined by a `*buildConfig*`, which is a REST object
+that can be used in a POST to the API server to create a new instance. The
+following example `*buildConfig*` results in a new build every time a Docker
+image tag or the source code changes:
```

Comment:
@bparees OK with the above changes? I basically just wanted to spell out "build configuration" at least once early on.